mod/message.php

```diff
@@ -165,11 +165,23 @@ function message_content(App $a)
 
 		$cmd = $a->argv[1];
 		if ($cmd === 'drop') {
+			$r = DBA::SelectFirst('mail', ['convid'], ['id' => $a->argv[2], 'uid' => local_user()]);
+			if(!DBA::isResult($r)){
+				info(L10n::t('Conversation not founded.') . EOL);
+				goaway('/message');
+			}
+
 			if (DBA::delete('mail', ['id' => $a->argv[2], 'uid' => local_user()])) {
 				info(L10n::t('Message deleted.') . EOL);
 			}
 
-			goaway('/message' );
+			$rr = DBA::SelectFirst('mail', ['id'], ['convid' => $r['convid'], 'uid' => local_user()]);
```

Comment:
The exact method name is `selectFirst` without the first letter capitalized.
Additionally, please use a meaning name for the variable. In this case, it seems to be the parent conversation, so I would use `$conversation` instead of `$rr`.

Reply:
Okay, I'll change that. I still get confused because I look about other code parts written like that but I do unterstand your request.

Reply:
We have masses of old and ugly code. But when we write new stuff, we want to raise the quality.